app.rb

```diff
@@ -180,6 +180,36 @@ class << self
     CategoryContribution.all.to_json
   end
 
+  get '/api/payments/?:id?' do |id|
+    # TODO: Figure out how to cache this
+    # cache_control :public
+    # last_modified Import.last.import_time
```

Comment:
This was copied from the /api/contributions entry point, can that one be removed too?